### PR TITLE
[css-fonts-5] Various Edits

### PR DESCRIPTION
**Pull request by fantasai**
Edits for the various metrics descriptors. Commit-by-commit review recommended. (It's not separate PRs because they build on each other. Please don't make me submit them one by one, I'll end up tangling all the branches in git...)

### CODE REVIEW

**Comment by fantasai** [comment]
reminder: DO NOT SQUASH WHEN MERGING

**Review by svgeesus — COMMENTED** [review]
Hmm, collides with https://github.com/w3c/csswg-drafts/pull/6137

**Comment by fantasai** [comment]
@svgeesus Indeed. I guess it's up to you which set of edits you like better. :)

File: css-fonts-5/Overview.bs [hunk 1]
```diff
@@ -178,53 +177,52 @@ the 'size-adjust' descriptor</h3>
...
+	<a spec="CSS-INLINE-3">line gap metric</a>
+	of the font, respectively.
+
+	<dl dfn-for="ascent-override!!descriptor,descent-override!!descriptor,line-gap-override!!descriptor" dfn-type=value>
+		<dt><dfn>normal</dfn>
+		<dd>
+			The corresponding metric value is obtained from the font as usual,
+			as if this descriptor were absent from the <code>@font-face</code> block.
+
+			Note: Since there are sources of such metrics in some font formats,
```
Comment (jfkthame) [inline css-fonts-5/Overview.bs:213]: Insert "multiple" before "sources", I think.

Comment (fantasai) [reply css-fonts-5/Overview.bs]: Done! Thanks for catching that. :)

**Comment by fantasai** [comment]
Ok, fixed the merge conflicts. Someone please decide whether or not this is acceptable?????

**Review by svgeesus — APPROVED** [review]

**Pull request merged**